src/components/EmptyStateComponent/index.tsx

```diff
@@ -35,6 +37,7 @@ function EmptyStateComponent({
     const styles = useThemeStyles();
     const [videoAspectRatio, setVideoAspectRatio] = useState(VIDEO_ASPECT_RATIO);
     const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const doesSubtitleContainCustomEmoji = containsCustomEmoji(subtitle ?? '') && !containsOnlyCustomEmoji(subtitle ?? '');
```

Comment:
Should we name this something like this, since this var will be `false` if the subtitle is only a custom emoji?

```suggestion
    const doesSubtitleContainCustomEmojiAndMore = containsCustomEmoji(subtitle ?? '') && !containsOnlyCustomEmoji(subtitle ?? '');
```

Reply:
That’s a good one, We should definitely use `doesSubtitleContainCustomEmojiAndMore` since it makes the purpose clearer.